synapse/tests/test_runtime.py

```diff
@@ -80,8 +80,21 @@ def runStormBasics(self, core):
         self.sorteq(core.eval('inet:ipv4="127.0.0.1" inet:ipv4->inet:dns:a:ipv4 inet:dns:a:fqdn->inet:fqdn'), [t6, t7])
 
         # test join operator basics
+        self.raises(BadSyntaxError, core.eval, 'inet:ipv4="127.0.0.1" join()')
```

Comment:
We should add tests that use the join() macro syntax <-  ( macro args should be the same as pivot )